app/IQ/IQClient.php

```diff
@@ -0,0 +1,136 @@
+<?php
+namespace App\IQ;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\RequestOptions;
+
+class IQClient
+{
+    /**
+     * @var Client
+     */
+    private $client;
+
+    private $username;
+
+    private $token;
+
+    /**
+     * The passed in value must be a Guzzle Client
+     *
+     * @param Client|null $client
+     */
+    public function __construct(
+        $client = null,
+        $host = 'http://localhost:8070/',
+        $username = 'admin',
+        $token = 'admin123',
+        $stage = 'develop'
```

Comment:
To my clueless eyes, it appears this field is not used, but instead the `$iq_client->submit_sbom(..., stage)` is used? Maybe one of these should be removed.

Reply:
Addressed in: dd0b2b7